test/Compiler.Dynamic.Tests/assets/test12/master_property_set.aspx

```diff
@@ -0,0 +1,62 @@
+<%@ Page Title="Home Page" Language="C#" AutoEventWireup="false" MasterPageFile="~/Masters/Default.Master" CodeBehind="master_property_set.aspx.cs" Inherits="SystemWebUISample.Pages.MasterPropertySetPage" %>
```

Comment:
This test seems to have a lot more than what you need to test. Can you pare it down to just a simple page referencing a master page in a different path?